src/SFML/Audio/MiniaudioUtils.cpp

```diff
@@ -25,81 +25,59 @@
 ////////////////////////////////////////////////////////////
 // Headers
 ////////////////////////////////////////////////////////////
+#include <SFML/Audio/AudioDevice.hpp>
 #include <SFML/Audio/MiniaudioUtils.hpp>
 #include <SFML/Audio/SoundChannel.hpp>
 
-#include <SFML/System/Angle.hpp>
 #include <SFML/System/Err.hpp>
 #include <SFML/System/Time.hpp>
 
 #include <miniaudio.h>
 
-#include <functional>
-#include <limits>
 #include <ostream>
 
 #include <cassert>
+#include <cstring>
 
 
 namespace sf::priv
 {
 namespace
 {
```

Comment:
Why is there an anonymous namespace inside of `sf::priv`? That's not typically how we do things.